packages/analytics-ma-browser/package.json

```diff
@@ -0,0 +1,66 @@
+{
+  "name": "@amplitude/analytics-ma-browser",
+  "version": "0.0.0",
```

Comment:
Should this be 1.0.0?

Reply:
We are going to update the version number before release

Reply:
I think we should start with `0.0.1`. And if we do, we should leave this PR at `0.0.0` and let the release bump it.